samples/pmic/native/npm2100_fuel_gauge/src/fuel_gauge.c

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdlib.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/drivers/sensor/npm2100_vbat.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/sys/util.h>
+
+#include "fuel_gauge.h"
+
+#include <nrf_fuel_gauge.h>
+
+static int64_t ref_time;
+
+static const struct battery_model_primary battery_models[] = {
+	[BATTERY_TYPE_ALKALINE_AA] = {
+		#include <battery_models/primary_cell/AA_Alkaline.inc>
+	},
+	[BATTERY_TYPE_ALKALINE_AAA] = {
+		#include <battery_models/primary_cell/AAA_Alkaline.inc>
+	},
+	[BATTERY_TYPE_ALKALINE_LR44] = {
+		#include <battery_models/primary_cell/LR44.inc>
+	},
+	[BATTERY_TYPE_LITHIUM_CR2032] = {
+		#include <battery_models/primary_cell/CR2032.inc>
+	},
```

Comment:
not an issue but style, cr2032 comes first in the .h, shouldn't it be ordered the same here?